extensions-core/druid-basic-security/src/main/java/org/apache/druid/security/basic/authorization/endpoint/BasicAuthorizerResource.java

```diff
@@ -665,7 +673,11 @@ public Response authorizerGroupMappingUpdateListener(
   )
   {
     authValidator.validateAuthorizerName(authorizerName);
-    return resourceHandler.authorizerGroupMappingUpdateListener(authorizerName, serializedGroupMappingAndRoleMap);
+
+    final Response response = resourceHandler.authorizerGroupMappingUpdateListener(authorizerName, serializedGroupMappingAndRoleMap);
+    performAuditIfSuccess(authorizerName, req, response, "Update group mappings for authorizer[%s]", authorizerName);
```

Comment:
Looking at this, I think we should include the authorizer name in the other audit messages in this class too. We can do it in a follow up PR or this one itself.